java/src/main/java/io/grpc/examples/wallet/AccountServer.java

```diff
@@ -82,12 +85,17 @@ void parseArgs(String[] args) {
               + "\n  --hostname_suffix=STR  Suffix to append to hostname in response header. "
               + "Default \""
               + s.hostnameSuffix
-              + "\"");
+              + "\""
+              + "\n  --observability_project=STR GCP project. If set, metrics and traces will be "
+              + "sent to Stackdriver. Default \"" + s.observabilityProject + "\"");
       System.exit(1);
     }
   }
 
   private void start() throws IOException {
+    if (observabilityProject != "") {
```

Comment:
nit: `!observabilityProject.isEmpty()`. Same for the client and wallet server.

Reply:
Done